### DISCUSSION TITLE
Zooming and panning: is their support for these operations

### DISCUSSION THREAD

**Discussion by hmf** [HanSolo/charts#66] [Q&A]
I would like to use this library for exploratory data analysis using Java (Scala). I would like to interactively and automatically pan and zoom a (time series) plot. I used the `TimeAxisTest` and it seems this is not possible (or is deactivated).  

Any suggestions on how I can implement this? 

TIA

**Comment by hmf** [comment]
I have just seen [issue 8](https://github.com/HanSolo/charts/issues/8).  8-(

**Comment by HanSolo** [comment]
Nope it‘s not implemented yet…and it needs some time to get it right

**Reply by hmf** [reply]
Yep. I am assuming that to support all (or most) plots it should be pretty general. Thanks.
